sde_collections/admin.py

```diff
@@ -6,7 +6,21 @@
 from .models.candidate_url import CandidateURL, ResolvedTitle
 from .models.collection import Collection, WorkflowHistory
 from .models.pattern import DivisionPattern, IncludePattern, TitlePattern
-from .tasks import import_candidate_urls_from_api
+from .tasks import fetch_and_update_full_text, import_candidate_urls_from_api
+
+
+@admin.action(description="Import candidate URLs from LRM Dev Server with Full Text")
+def fetch_full_text_lrm_dev_action(modeladmin, request, queryset):
```

Comment:
This looks like the same function repeated twice. Can we make it dry so that they at least call a function which takes the server name as an arg?

Reply:
Changes made, Each admin action calls a shared function with the server name as an argument, ensuring code reusability while maintaining "clear options" in the admin interface.